R/cluster.R

```diff
@@ -154,8 +152,27 @@ makeCluster <-
       installAndStartContainerCommand
     )
 
+    commandLine <- dockerInstallCommand
+
+    # log into private registry if registry credentials were provided
+    if (!is.null(config$dockerAuthentication) &&
+        !is.null(config$dockerAuthentication$username)) {
+
+      username <- config$dockerAuthentication$username
+      password <- config$dockerAuthentication$password
+      registry <- config$dockerAuthentication$registry
+
+      # TODO: Use --password-stdin when logging in to not show the password on the command line
```

Comment:
It current shows the docker user password in the console?

Reply:
it shows it as part of the setup scripts since we run the command (i.e. it is part of the start task command line). I'll look into removing it now.